internal/sdkv2provider/resource_cloudflare_account.go

```diff
@@ -66,8 +66,7 @@ func resourceCloudflareAccountRead(ctx context.Context, d *schema.ResourceData,
 
 	foundAcc, _, err := client.Account(ctx, accountID)
 	if err != nil {
-		var notFoundError *cloudflare.NotFoundError
-		if errors.As(err, &notFoundError) {
+		if errors.As(err, &cloudflare.NotFoundError{}) {
```

Comment:
i can't recall what version of Go i had this issue in but it was required to have an initialised struct for `errors.As` -- using it in this fashion caused it to never match.

Reply:
No problem, have backed out this part of the change.